lib/commands/update.js

```diff
@@ -195,6 +197,24 @@ class UpdateCommand extends Command {
         });
     }
 
+    rollbackFromFail(error, newVer, force = false) {
```

Comment:
For v1 blogs we don't execute `knex-migrator rollback` yet. We should probably skip this function if you upgrade from 1.24 to 1.25 and it fails.